tests/test_integrations/test_rapid7.py

```
@@ -29,31 +28,25 @@ def test_integration_active_invalid_key(self, sample_integration_dict):
 
     def test_empty_credentials(self, sample_integration_dict):
         tokens = {"region": "us"}
-        integration = sample_integration_dict("rapid7", tokens)
-        service = integration_service_factory.get_service(None, integration)
-        res = service.is_active()
-        assert not res["success"]
-        assert "No credentials configured" in res["error"]
+        with pytest.raises(Exception):
```

Ruff (0.16.0)

[warning] Do not assert blind exception: `Exception`

(B017)